.github/workflows/gatsby.yml

```diff
@@ -20,6 +20,8 @@ jobs:
 
       - run: yarn install
 
+      - run: yarn test
+
       - run: yarn format
 
       - run: yarn lint
```

Comment:
myslim si ze to ne vadi kde je to, ale ponechal placeholder na 32 line. Jestli myslis nechat tady, odstrani prosim 32 line

Reply:
jejda, toho jsem si nevšiml, díky, opravím